spec/quality_spec.rb

```diff
@@ -223,6 +223,11 @@ def check_for_specific_pronouns(filename)
         if Bundler.rubygems.provides?(">= 2.4")
```

Comment:
This has been removed from latest master, that why the PR shows conflicts, I think.